libcni/api.go

```diff
@@ -175,6 +181,15 @@ func (c *CNIConfig) ensureExec() invoke.Exec {
 	return c.exec
 }
 
+type cachedInfo struct {
+	Kind           string                 `json:"kind"`
+	Config         []byte                 `json:"config"`
+	CniArgs        [][2]string            `json:"cniArgs,omitempty"`
+	CapabilityArgs map[string]interface{} `json:"capabilityArgs,omitempty"`
+	RawResults     map[string]interface{} `json:"results,omitempty"`
```

Comment:
@mccv1r0 let's make this `RawResult` to stay consistent. There's really only one Result object here.